tools/scripts/aducm.mk

```diff
@@ -273,6 +273,8 @@ copy_pinmux:
 
 update: copy_pinmux
 
+$(PLATFORM)_reset:
```

Comment:
make sure these empty ones don't generate make warnings for being empty, i recall seeing some empty rule warnings at some point but i'm not sure